app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -1,7 +1,6 @@
 module Articles
   module Feeds
     class LargeForemExperimental
-      include FieldTest::Helpers
```

Comment:
This was a leftover from the A/B tests and is no longer needed.

Reply:
Seems cool to me as a matter of refactoring, just want to comment that the intention of the experiments was a matter of keeping the testing code around and change the _contents_ of the tests every 2 weeks.

Definitely ways to clean up the code and do the whole thing in a different way, but just felt like mentioning this in case there was any confusion about the original intent.

Reply:
Yeah, I figured, but tbh I don't necessarily think this is the best way forward. 

For example: some things in the basic feeds controller are hardcoded to use methods from `LargeForemExperimental`. That is not intuitive and probably means that these things should go into a separate feed utilities service object or something. I'd also strongly prefer if new experiments leave old code alone which reduces the chances of accidentally breaking something that already worked. If someone wants to add a new experiment, they can add a new class, if it's very similar to an existing experiment they can always inherit from that and only override what they need.

My main intent here is to be able to better distinguish between code that is now permanent (e.g. a feed strategy that will stick around) or temporary (an experiment that may be removed in a couple of weeks).

Ideally I'd want to rename this class to "LargeForem" eventually. New experiments go into new classes and once we have a clear winner we can decide if we either make it a complete new and alternative strategy or update an existing one. Does that make sense?

Reply:
Cool cool, makes sense. I defer to your vision here.